bigquery/indexer.py

```diff
@@ -5,7 +5,6 @@
 import os
```

Comment:
We do e2e sanity checking [here](https://github.com/DataBiosphere/data-explorer-indexers/blob/00a8aedd8a89e41d32f4b966f71ee0d9ec0a0ec2/.circleci/config.yml#L18).

Can you add a "Run the GCS indexer" section? Let's just assert that the index has 2559 documents. (You're welcome to add more extensive tests if you want.) Let's also delete the index "curl -XDELETE localhost:9200/1000_genomes" at the end of BQ/GCS sections.

Reply:
Ah sorry didn't see that CI testing, added GCS indexing section.